exchange/adapter_builders.go

```diff
@@ -352,7 +352,6 @@ func newAdapterBuilders() map[openrtb_ext.BidderName]adapters.Builder {
 		openrtb_ext.BidderUnicorn:           unicorn.Builder,
 		openrtb_ext.BidderUnruly:            unruly.Builder,
 		openrtb_ext.BidderValueImpression:   apacdex.Builder,
-		openrtb_ext.BidderVerizonMedia:      yahoossp.Builder,
```

Comment:
Can you please confirm if you want to remove this alias or update it to use `yahooAdvertising.Builder`? As in the PR description it is mentioned that `The yahoossp alias should be supported.`